.vscode/settings.json

```diff
@@ -7,9 +7,12 @@
   "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
   "typescript.tsdk": "node_modules/typescript/lib",
   "eslint.workingDirectories": [
+    "docs",
     "examples/appdir",
```

Comment:
are wildcards supported here? if so, `examples/*` and `packages/*` so we don't have to worry about this again

Reply:
I'll check